static/app/components/events/suspectCommits.tsx

```diff
@@ -80,7 +93,34 @@ export function SuspectCommits({group, eventId, project, commitRow: CommitRow}:
 
   const commitHeading = tn('Suspect Commit', 'Suspect Commits (%s)', commits.length);
 
-  return (
+  const firstCommit = commits[0];
```

Comment:
Hmm, I wonder we show the rest of the commits without falling back to the old UI. You may have to follow up with @vuluongj20

Reply:
showing the rest of the commits will be handled in a future PR, the scope for this PR is limited to just showing the first commit, this fallback is a bug that ill fix